templates/forms/UserFormsCheckboxSetField.ss

```diff
@@ -0,0 +1,11 @@
+<% if $Options.Count %>
+    <% loop $Options %>
+        <div class="$Class">
+            <input id="$ID" class="checkbox" name="$Name" type="checkbox" value="$Value.ATT"<% if $isChecked %>
```

Comment:
No ability to add extra CSS to this field - is that deliberate? I suppose they'd get added to the holder instead?